_pages/2018/x/psets/4/resize/more/resize.adoc

```diff
@@ -44,7 +44,7 @@ Implement a program called `resize` that resizes (i.e., enlarges or shrinks) 24-
 * Your program should accept exactly three command-line arguments, whereby
 +
 --
-** the first (`f`) must be a floating-point value in (0.0, 100.0],
+** the first (`f`) must be a floating-point value in (0.0, 100.0),
```

Comment:
Pretty sure the `]` was deliberate: means "inclusive"!

Reply:
ah! TIL.